clients/upload.go

```diff
@@ -0,0 +1,64 @@
+package clients
```

Comment:
Would suggest calling this file something else like `filecopy.go` so that it's not clashing with `handlers/upload.go`. It would be a little less confusing this way.

Reply:
Ah good point, will do